Jetsnack/app/src/main/java/com/example/jetsnack/ui/home/cart/Cart.kt

```diff
@@ -150,12 +160,115 @@ private fun CartContent(
             )
         }
         items(orderLines) { orderLine ->
-            CartItem(
-                orderLine = orderLine,
-                removeSnack = removeSnack,
-                increaseItemCount = increaseItemCount,
-                decreaseItemCount = decreaseItemCount,
-                onSnackClick = onSnackClick
+            SwipeDismissItem(
+                background = { offsetX ->
+                    // Adjust padding right for the main Row, while the delete looks like an Icon the padding must be different from zero
+                    val paddingRight = if (offsetX < -15.dp && offsetX > -200.dp) {
+                        ((offsetX / -2))
+                    } else {
+                        0.dp
+                    }
+                    // Row background is uiFloated until the Remove item text appears, then it is replaced with a red Background
+                    val background = if (offsetX >= -200.dp) {
```

Comment:
200.dp seems a bit arbitrary? Is this a computed value?

Reply:
This value is no longer used, however I do have some values for changing the behavior from a circular red shape to a full red background, do you have another suggestion on how to make this change instead of using fixed offset breakpoints?